### PR TITLE
Docs/update and add timelines

### PR DESCRIPTION
**Pull request by ther3sa**
- Added Timeline 25.06
- Updated Timeline 24.12 and  25.03 to better display Tractus-X meeting
- Added standard timeline in main timeline 
- Made png zoomable 

Bildschirmfoto 2024-11-25 um 13 26 20

Bildschirmfoto 2024-11-25 um 13 26 53

Bildschirmfoto 2024-11-25 um 13 27 05

Bildschirmfoto 2024-11-25 um 13 27 12

### CODE REVIEW

File: src/pages/timelines.md [hunk 1]
```diff
@@ -16,8 +17,17 @@ Milestones featuring the Catena-X logo indicate closed meetings, which are organ
 
 ## Catena-X Io, incl. Eclipse Tractus-X Release 24.12 & 25.03
```
Comment (stephanbcbauer) [inline src/pages/timelines.md]:
```suggestion
### Catena-X Io, incl. Eclipse Tractus-X Release 24.12 & 25.03
```

File: src/pages/timelines.md [hunk 2]
```diff
@@ -16,8 +17,17 @@ Milestones featuring the Catena-X logo indicate closed meetings, which are organ
...
-![Timeline for Release 24.12](@site/static/img/cx-timeline-24.12.png)  
+[![Timeline for Release 24.12](@site/static/img/cx-timeline-24.12.png)](@site/static/img/cx-timeline-24.12.png)
 *Timeline for Release 24.12*
 
-![Timeline for Release 25.03](@site//static/img/cx-timeline-25.03.png)
+[![Timeline for Release 25.03](@site/static/img/cx-timeline-25.03.png)](@site/static/img/cx-timeline-25.03.png)
 *Timeline for Release 25.03*
+
+[![Timeline for Release 25.06](@site//static/img/cx-timeline-25.06.png)](@site//static/img/cx-timeline-25.06.png)
+*Timeline for Release 25.06*
```
Comment (stephanbcbauer) [inline src/pages/timelines.md:29]:
```suggestion

### Catena-X Saturn, incl. Eclipse Tractus-X Release 25.06 & 25.09

[![Timeline for Release 25.06](@site//static/img/cx-timeline-25.06.png)](@site//static/img/cx-timeline-25.06.png)
*Timeline for Release 25.06*
```

File: src/pages/timelines.md [hunk 3]
```diff
@@ -16,8 +17,17 @@ Milestones featuring the Catena-X logo indicate closed meetings, which are organ
...
+
+[![Timeline for Release 25.09](@site/static/img/cx-timeline-25.09.png)](@site/static/img/cx-timeline-25.09.png)
+*Timeline for Release 25.09*
+
+[![Timeline for Release 25.12](@site/static/img/cx-timeline-25.12.png)](@site/static/img/cx-timeline-25.12.png)
+*Timeline for Release 25.12*
```
Comment (stephanbcbauer) [inline src/pages/timelines.md:37]:
```suggestion
### Catena-X Titan, incl. Eclipse Tractus-X Release 25.12 & 26.03

[![Timeline for Release 25.12](@site/static/img/cx-timeline-25.12.png)](@site/static/img/cx-timeline-25.12.png)
*Timeline for Release 25.12*
```

Comment (stephanbcbauer) [inline src/pages/timelines.md:37]:
```suggestion
[![Timeline for Release 25.12](@site/static/img/cx-timeline-25.12.png)](@site/static/img/cx-timeline-25.12.png)
*Timeline for Release 25.12*

:::note
The timeline for Release 26.03 will be published soon.
:::
```

**Review by stephanbcbauer — COMMENTED** [review]
Like the PR, just solved some linting issues. I Also added some suggestions. The idea would be to add headlines on the timelines. It's more structures and the headlines can be used for linking.

Would like this:

image
image
image
image

**Comment by stephanbcbauer** [comment]
BTW I think on the picture we have to look carefully, sometimes the deployment on XX.XX was wrong.

**Comment by ther3sa** [comment]
> BTW I think on the picture we have to look carefully, sometimes the deployment on XX.XX was wrong.

Thanks for the tip. I'll check it and provide an update :)

File: src/pages/release-management.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ Effective management of versions and changes for Catena-X standards, KITs and op
 
 ## Support Dates
 
-![Support Dates](/img/support-dates.jpg)
+[![Support Dates](/img/support-dates.jpg)](/img/support-dates.jpg)
```
Comment (stephanbcbauer) [inline src/pages/release-management.md]: Even if /img/... works, you should work with @site/static/img/ to indicate, that the images is really a static image and not in the current folder in an img folder

File: src/pages/release-management.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ Based on the first experiences in live operation, the release management for the
 
 ## Rationale for Changes
 
-![Release Timeline](/img/release-schedule.png)
+[![Release Timeline](/img/release-schedule.png)](/img/release-schedule.png)
```
Comment (stephanbcbauer) [inline src/pages/release-management.md]: same here

**Review by stephanbcbauer — CHANGES_REQUESTED** [review]
Last change request :)

**Review by stephanbcbauer — APPROVED** [review]
LGTM

**Pull request merged**